feature/auth/src/main/res/values/strings.xml

```diff
@@ -18,4 +18,5 @@
     <string name="feature_auth_error_password_length">Invalid password length</string>
     <string name="feature_auth_error_not_connected_internet">No Internet Connection</string>
     <string name="feature_auth_error_login_failed">Login Failed, Please Try Again Later.</string>
+    <string name="feature_auth_error_unexpected">Unexpected Error Please Enter the valid Login details</string>
```

Comment:
Would you mind changing the string name to feature_auth_error_wrong_credentials ???

Reply:
Sure Sir

Reply:
@Darkeye14  I made the required changes may you please review it